README.md

```diff
@@ -17,66 +17,52 @@
 ### Running parameters
```

Comment:
I think it would be better to have an example instead of listing all the parameters in how-to-use block.